neoload/commands/test_results.py

```diff
@@ -201,6 +201,13 @@ def get_end_point(id_test: str = None, operation=''):
     slash_id_test = '' if id_test is None else '/' + id_test
     return rest_crud.base_endpoint_with_workspace() + __endpoint + slash_id_test + operation
 
+def prompt_is_locked(field):
+    is_locked = tools.string_to_bool_json(input(field))
+    while is_locked is None :
+        print("\n (Accepted value for true : )", tools.get_true_values())
```

Comment:
Typo: add a "s" to "accepted value**s**"
Remove parenthesis ?
Don't add space between "true" and ":" => In English, never add a space before ? ! :   In French, ALWAYS add a space before ? ! :    Mdr^^